code/modules/mob/living/carbon/human/species/zombie.dm

```diff
@@ -53,8 +53,8 @@
 	if(zombie.glasses) zombie.drop_inv_item_on_ground(zombie.glasses, FALSE, TRUE)
 	if(zombie.wear_mask) zombie.drop_inv_item_on_ground(zombie.wear_mask, FALSE, TRUE)
 
-	if(zombie.lying)
-		zombie.lying = FALSE
+//	if(zombie.lying)
+//		zombie.lying = FALSE
```

Comment:
It doesn't make sense to have this and especially not in handle_post_spawn because it's not a value you assign but one that gets set by update_canmove on Life tick